tests/test_api.py

```diff
@@ -17,25 +17,29 @@
 SECURITY_CODE = "123456"
 PHONE_NUMBER = "+13478379634"
 SESSION_TOKEN = "phone-auth-session-token"
+BACKEND_SERVICES = [
+    ("twilio.rest.Client", "phone_verify.backends.twilio.TwilioBackend"),
+    ("kavenegar.KavenegarAPI", 'phone_verify.backends.kavenegar.KavenegarBackend')
+]
 
 
 def test_phone_registration_sends_message(client, mocker):
     url = reverse("phone-register")
     phone_number = PHONE_NUMBER
     data = {"phone_number": phone_number}
-    twilio_api = mocker.patch(
-        "phone_verify.services.PhoneVerificationService.send_verification"
-    )
-
-    response = client.post(url, data)
 
-    assert response.status_code == 200
-    assert twilio_api.called
-    assert "session_token" in response.data
-    SMSVerification = apps.get_model("phone_verify", "SMSVerification")
-    assert SMSVerification.objects.get(
-        session_token=response.data["session_token"], phone_number=phone_number
-    )
+    for backend_service, backend in BACKEND_SERVICES:
```

Comment:
Let's flip the backend and backend_service defined in `BACKEND_SERVICES`

So, it will be:

```
BACKEND_SERVICES = [
    ("phone_verify.backends.twilio.TwilioBackend", "twilio.rest.Client"),
    ('phone_verify.backends.kavenegar.KavenegarBackend', "kavenegar.KavenegarAPI")
]
```